### ISSUE TITLE
Can't log into my own account

### ISSUE THREAD

**Issue by joesh1** [mastodon/mastodon#18162]
### Steps to reproduce the problem

1. Go to Log On page
2. Put in email and password
3. Error message: Invalid Address or Password
...

I've seen posting this elsewhere, but I'll repeat it. 

I can't sign into my account. When I go to the log on page it says 'Invalid Address or Password'. I know this is not correct, as I can use the confirmation email I've saved which automatically sends me to my page/account.

Secondly, I've checked my saved password in my browser (Opera) and the password I've used is correct. Lastly, just to add that I've tried several browsers, but the problem is the same.   

### Expected behaviour

I should have gone to my account

### Actual behaviour

Error message

### Specifications

Browser: Opera

OS: Mac Os 11.6.5

**Comment by ClearlyClaire** [comment]
Are you perhaps trying to log into the wrong server? Mastodon has multiple servers that interconnect, but you need to log into the server you registered with, not the other ones.

**Comment by joesh1** [comment]
Hi, sorry I didn't see you message. 

So, to answer your question, I am trying to login on my server page. I'm on Mastodon Social, but I just can't login on the page. As I've already said, I use an old link in my welcoming mail from Mastodon in my email folder to go to my account (which is odd, as that works). However, if I go to the login page - which I thought was useful to use (especially if you want to save time going back to your email to then go to your account) - I get an error message. I few weeks ago I decided that I should maybe just delete my account and start again, which I did. However, that didn't solve the problem. 

Today, for an experiment, I thought I should change my password (from within my Mastodon account, to be sure). I received the confirmation mail from Mastodon saying my password had changed - all good up until here. Next, I then went to the Login page and tried to login ..... same problem, "Invalid Address or Password"!

So, any suggestions out there in GitHub/Mastodon land. 

Oh, and by the way, I can't get onto the Mastodon forum either to ask the question as they're not accepting new subscribers (at present).

**Comment by ClearlyClaire** [comment]
This still sounds like you've registered to a different server than you are trying to log into. When clicking on the mail you've received, take note of the domain name in the browser's address bar. Compare it to the one you are trying to log into.

> Oh, and by the way, I can't get onto the Mastodon forum either to ask the question as they're not accepting new subscribers (at present).

Are you talking about https://discourse.joinmastodon.org/ ? As explained in the banner, discussions have moved to https://github.com/mastodon/mastodon/discussions

**Comment by joesh1** [comment]
No, I've tried that a dozen times and it's definitely the right page (browser - mastodon.social/auth/sign_in). Any other suggestions?

As for the forum, indeed you're correct, I hadn't seen that discussions had moved.

**Comment by ClearlyClaire** [comment]
I'm not sure what to suggest now. Can you share your profile page? Click the link that allows you to log in, and right-click on your avatar in the Web UI to get the link.

**Comment by joesh1** [comment]
Thanks Claire, I took a different route which was to delete the account. The whole thing was most odd. I could change my password, from inside my profile, then receive a mail saying my password/email whatever I wanted, had changed. However, when I simply asked for another password ... I never received a reset mail. Very odd indeed. 

Next, to test everything I created a new profile (using a different email) and then tried logging in. That worked, so what's happening in the Mastodon server who knows. 

So, as I said, I deleted all the accounts, and now I'm going to wait a few days to let my email and other data be erased from the system and then start again and see what happen - I hadn't started getting many followers, and I know the few I did have.  

Keep an eye here and if you don't see me back again within a week ... it means everything is working as normal. 

Big thanks for your help anyhow.

**Comment by ncs-pl** [comment]
I know this issue was closed but it is still happening fyi

**Comment by ClearlyClaire** [comment]
@ncs-pl hi! We will need more information in order for us to help you. This is also something that is more likely to have to do with your server admins than the Mastodon project itself.
